lib/fog/ovirt/requests/compute/v4/update_vm.rb

```diff
@@ -0,0 +1,48 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        class Real
+          # rubocop:disable Metrics/AbcSize
+          def update_vm(attrs)
+            attrs = attrs.dup
+            vm_service = client.system_service.vms_service.vm_service(attrs[:id])
+
+            if attrs[:cores].present?
+              cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => "1")
+              attrs[:cpu] = OvirtSDK4::Cpu.new(:topology => cpu_topology)
+            end
+            attrs[:cluster] = if attrs[:cluster].present? ? OvirtSDK4::Cluster.new(:id => attrs[:cluster]) : nil
```

Comment:
the `if` is causing issues, as it's missing `end`, and leads to syntax error, it should be just removed.

Reply:
yeah it was an `if` copied from before, this needs to change to delete the attribute anyway instead of returning nil